### DISCUSSION TITLE
install location

### DISCUSSION THREAD

**Discussion by ederson13** [NativePHP/laravel#328] [Q&A]
Currently (unless i m missing something) the app is installed in AppData\Local\Programs\laravel\AppName 

in there any to way to change this path ?

**Comment by simonhamp** [comment]
That's not where it's installed, only where various app data is kept. The name of the folder should change based on the config in the production build. If it doesn't work as intended, please can you [raise an issue](https://github.com/NativePHP/laravel/issues/new/choose)?
